otel.yml

```diff
@@ -21,6 +21,9 @@ extensions:
     check_interval: 1s
     limit_mib: 700
     spike_limit_mib: 180
+  health_check:
+    endpoint: "localhost:13133"
+    path: "/health/status"
```

Comment:
The default value here, according to https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/extension/healthcheckextension/README.md#health-check, is `/`.  But I think it's better to use a more specific path for the health check endpoint.  Again, happy to change this if others have a better suggestion.

Reply:
Maybe `/liveness`? In that sense, we will be more aligned to the `elastic-agent` health check endpoint https://github.com/elastic/elastic-agent/pull/4499

Reply:
Done in 6f2306c6c8.  Thanks for the suggestion.

Reply:
Just to give more context on the Helm configuration, any change on the default health_check configuration will need to be overrided in the `liveness and readinesss` proves of the Helm Chart config too:

https://github.com/open-telemetry/opentelemetry-helm-charts/blob/main/charts/opentelemetry-collector/values.yaml#L395

```yaml
livenessProbe:
  httpGet:
    port: 13133
    path: /health/status
readinessProbe:
  httpGet:
    port: 13133
    path: /health/status
```

Reply:
Thanks @rogercoll.  

Does this mean we will need to specify additional `--set` arguments to `helm chart install` for the configuration keys you mentioned above, assuming we are using the bundled `otel.yml` file via `--set command.name="/usr/share/elastic-agent/elastic-agent" --set command.extraArgs="{otel, --config=otel.yml}"`?  

So something like `--set livenessProbe.httpGet.path=/liveness --set readinessProbe.httpGet.path=/liveness`?

Sorry if this is a basic question but I'm a n00b when it comes to Helm.

Reply:
Correct, you will need to override the probes path with the --set flag. But as I was explaining in this [comment](https://github.com/elastic/elastic-agent/pull/5369#issuecomment-2314505406), the upstream Helm Charts override the arguments launched within the container and specify their custom configuration args. That means that the `otel.yml` file will not be taken into consideration and a custom configuration should be provided with a K8s config map.

I am not 100% sure about changing the default `health_check` path in the sample configuration, maybe @elastic/elastic-agent-control-plane can share their feedback based on the current elastic-agent health endpoint.

Reply:
@blakerouse @andrzej-stencel @swiatekm do you have any opinions here?

Reply:
The elastic-agent uses `/liveness` as the default endpoint for its own healthcheck. I don't know if we want to use the same for OTEL, as that would change the behavior of the default collector and probably don't want to do that.

I would like to also see v2 health check being used at some point, as it does a much better job of actually providing the true health of the collector.

Reply:
Thanks @blakerouse. I will revert the `path` here in this PR to `/health/status` again for the reason you mentioned. 

Agreed on switching to healthcheckv2 in a future PR :)